ledger/controllers/grants.js

```diff
@@ -616,6 +734,26 @@ const uploadGrants = (protocolVersion) => (runtime) => {
   }
 }
 
+v1.create =
+{ handler: uploadGrants(1),
+
+  auth: {
+    strategy: 'session',
+    scope: [ 'ledger' ],
+    mode: 'required'
+  },
+
+  description: 'Create one or more grants',
+  tags: [ 'api' ],
+
+  validate: { payload: grantsUploadSchema },
+
+  payload: { output: 'data', maxBytes: 1024 * 1024 * 20 },
+
+  response:
+    { schema: Joi.object().length(0) }
+}
+
```

Comment:
This endpoint was removed in https://github.com/brave-intl/bat-ledger/pull/505, was that a mistake?

Reply:
can remove

Reply:
must have been a bad merge. can remove